spec/api/v1/admin/patients_spec.rb

```diff
@@ -0,0 +1,162 @@
+require 'rails_helper'
+require 'active_record'
+# rubocop:disable RSpec/MultipleExpectations
+RSpec.describe V1::Admin::Patients, type: :request do
+  let(:patients_crud_url) { '/api/v1/admin/patients' }
+  let(:patients_params) do
+    {
+      address: Faker::Address.full_address,
```

Comment:
I know that data that is generated randomly is the best choice for tests, but try to not overusing of `Faker`, since its performance is not good enough - a note for the future PRs